Doc/whatsnew/3.14.rst

```diff
@@ -1073,6 +1073,10 @@ Deprecated
   :c:macro:`!isfinite` available from :file:`math.h`
   since C99.  (Contributed by Sergey B Kirpichev in :gh:`119613`.)
 
++ :c:func:`!PySequence_In` function is :term:`soft deprecated`.
```

Comment:
- `PySequence_In` is documented, so we don't need to suppress the link with `!`
- IMO, it would be good to mention that the function doesn't appear in earlier documentation.
```suggestion
+ The previously undocumented function :c:func:`PySequence_In` is :term:`soft deprecated`.
```